src/olympia/activity/tasks.py

```diff
@@ -21,15 +21,19 @@ def process_email(message, spam_rating, **kwargs):
             if header.get('Name', '').lower() == 'message-id':
                 msg_id = header.get('Value')
     if not msg_id:
-        log.error('No MessageId in message, aborting.')
-        log.error(message)
```

Comment:
the only downside with combing the message content in with the error text is stackdriver has a habit of losing large log messages (try searching for "TextCharSet" in the same timeperiod)

Reply:
Do we care ? What can we do about this ? (can't find it in stackdriver atm for some reason)

Reply:
the reason is stackdriver loses large log messages 😉

Reply:
we probably don't care because we never look at _anything_ to do with email -> activitylog processing, but just pointing out we lose that message from stackdriver in the meantime.

Reply:
Ah, lol, I'm dumb :) Are you sure it's stackdriver ? Would that also happen with the message passed as `extra` ?

Reply:
> Are you sure it's stackdriver ? 

It's somewhere in the pipeline.  I noticed it last year when I was debugging static theme problems - the log entries are in the raw logs but missing in stackdriver (the entry is there but no message content so difficult to identify among the other noise).

> Would that also happen with the message passed as extra ?

no idea tbh.  Lets find out!